recipes/freebayes/build.sh

```diff
@@ -20,10 +20,18 @@ export INCLUDES="-I . -Ihtslib -I$PREFIX/include -Itabixpp/htslib -I\$(INC_DIR)
 export LIBPATH="-L. -Lhtslib -L$PREFIX/lib"
 export CXXFLAGS="-O3 -D_FILE_OFFSET_BITS=64 -std=c++0x"
 
+sed -i.bak 's/^CC.*//g' SeqLib/htslib/Makefile
+sed -i.bak 's/^CC.*//g' SeqLib/bwa/Makefile
+sed -i.bak 's/^CC.*//g' SeqLib/fermi-lite/Makefile
```

Comment:
Is there no way to get around vendoring these? These should all really be simple dependencies.

Reply:
As far as I understand it, and I'm way out of my coding comfort zone, SeqLib is a C++ implementation of HTSlib, bwa, and fermikit [https://github.com/walaj/SeqLib](https://github.com/walaj/SeqLib), and that FreeBayes needs these specific libraries.  Again, this isn't my program, and I didn't create this bioconda package initially.  I'm just trying to maintain it because it is a dependency of one of my packages.

Reply:
Do you need this right away? If not, I'd like to try and get seqlib built as its own package and then switch freebayes to using non-vendored dependencies.

Reply:
BTW, I enjoy your commit messages :)

Reply:
Nevermind, freeBayes statically links to everything anyway, so there's no point.